### DISCUSSION TITLE
EasyRPG support?

### DISCUSSION THREAD

**Discussion by danijiri2000** [Rparadise-Team/Koriki#20] [General]
Good morning or evening to whoever is reading this query, in the previous version of Koriki, in Retro Arch there was the option to select content directly by searching in the game files, which I used to play games that use the Easy RPG core (games like: YumeNikki). Currently in version 1.3 of Koriki, I can't find that option. In addition, games that use EasyRPG do not have a directory assigned to them (I don't have the knowledge to make my own). Is there any way to select them directly from retroarch?

I would also like to report a bug that I have noticed when playing with EasyRPG in Koriki version 1.2 (I don't know how much it applies to version 1.3) which is that when you use fastforward the sound of the game slows down and even when you stop it, it continues slowing down.

**Comment by Ninoh-FOX** [comment]
I understand that you loaded the games from the same retroarch?  If so, you have to go to the menu settings and activate the option to load cores and load contents.  You can put ROMS/EASYRPG in the folder and it should appear in the list.  I understand that the team has not enabled it visible from SM because of the complexity of rpg maker games.  Regarding the sound, one option that you have is to deactivate the sound in the advance or rewind from retroarch to see if this solves the sound problem.

**Comment by Ninoh-FOX** [comment]
we have reuploaded koriki 1.3 with some fixes, now you should be able to play rpg maker fine
